evaluator/evaluator.py

```diff
@@ -505,11 +506,11 @@ def process_message(message):
                     LOGGER.info("Received message type: %s", msg_dict['type'])
                     LOGGER.info("VMaaS metadata changed, need to re-evaluate all systems")
                     self._sync_cve_md()
-                    cur.execute("select inventory_id, vmaas_json from system_platform")
+                    cur.execute("select inventory_id from system_platform")
                     # reevaluate updates for every system in the DB
-                    for system_platform in cur.fetchall():
-                        self.evaluate_vmaas(system_platform)
-                elif msg_dict['type'] == 'upload_new_file':
+                    for inventory_id, in cur.fetchall():
```

Comment:
I like using inventory_id - there are a few other system_platform vars in evaluator, can we make it use inventory_id consistently?

Reply:
sure, I'll add it

Reply:
actually, on other places is not useful to rename it because:
1. both inventory_id and vmaas_json columns are selected
2. fetchone instead of fetchall is used => inventory_id can't be extracted from tuple like I did here because it can return None

Reply:
Ah, I see - thanks for checking!